vk_requests/utils.py

```diff
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 import logging
-from collections import Iterable
+try:
```

Comment:
Recently six also added a compatibility layer for this change. `from six.moves.collections_abc import Iterable` will work from six 1.13 and above : https://github.com/benjaminp/six/commit/0dc54ead62c7fb06edabf1ff22fb71c0c6d958e1

Reply:
@vol1ura could you update it and check if suggestions from @tirkarthi would work?

Reply:
Yes it works with `six` since 1.13.0.